htdocs/core/class/commonobject.class.php

```diff
@@ -5350,6 +5354,13 @@ public function printObjectLine($action, $line, $var, $num, $i, $dateSelector, $
 			// Output template part (modules that overwrite templates must declare this into descriptor)
 			// Use global variables + $dateSelector + $seller and $buyer
 			// Note: This is deprecated. If you need to overwrite the tpl file, use instead the hook printObjectLine and printObjectSubLine.
+
+			$qty_shipped = 0;
+			if (isset($this->expeditions[$line->id])) {
+				$qty_shipped = $this->expeditions[$line->id];
+			}
+			$disableedit = $qty_shipped >= $line->qty;
```

Comment:
May be the disableedit should be one line higher in the if ?

Reply:
I've rethought the logic and I don't think you should put it in the if, you do set qty_shipped to 0 outside the if and each line object has a quantity so everything is well defined, you have to recalculate $disableedit each time because it's a loop and each line is different.
Then there's a second solution where you actually put it in the loop and delete qty_shipped before the if, but to do this you have to redefine disableedit at each iteration, otherwise if it doesn't work in the if, it will take the disableedit value of the previous line.